keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -179,6 +180,12 @@ Additionally, this KEP introduces a new `clusterTrustBundle` kubelet projected
 volume source that gives workloads easy filesystem-based access to trust anchor
 sets that they might need.
 
+A new ClusterTrustBundle `kubernetes.io:kube-apiserver-server:live` also gets created
+for all clusters. In combination with the new projected volume source,
+this trust bundle can be eventually used to replace the `kube-root-ca.crt` configMaps
+that nowadays live in all namespaces. The trust bundle uses a new signer,
+`kubernetes.io/kube-apiserver-server`.
```

Comment:
nit: `kubernetes.io/kube-apiserver-server` reads funny to me... `kubernetes.io/kube-apiserver-serving` parallels `kubernetes.io/kubelet-serving` better